<launch>
  <node pkg="pcl_cloud_tools" type="extract_clusters_on_table_server" name="extract_clusters_server" output="screen" respawn="true">
    <!-- Parameters --> 
    <param name="z_max_limit" value="0.9"/>
    <param name="cluster_min_height" value="0.02"/>
    <param name="cluster_max_height" value="0.3"/>	
    <param name="object_cluster_tolerance" value="0.05"/>
    <param name="object_cluster_min_size" value="50"/>
    <param name="nr_cluster" value="1"/>
    <param name="downsample" value="true"/>
    <param name="voxel_size" value="0.01"/>
    <param name="save_to_files" value="true"/>
    <param name="point_cloud_topic" value="/camera/rgb/points"/>
    <param name="publish_token" value="false"/>	
 </node>
</launch>
